presentation-api/receiving-ua/PresentationConnection_send-manual.https.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Sending a message through PresentationConnection</title>
+<link rel="author" title="Tomoyuki Shimizu" href="https://github.com/tomoyukilabs/">
+<link rel="help" href="http://w3c.github.io/presentation-api/#sending-a-message-through-presentationconnection">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="common.js"></script>
+<script src="support/stash.js"></script>
+
+<p>Click the button below and select the available presentation display, to start the manual test.</p>
+<button id="presentBtn">Start Presentation Test</button>
+
+<script>
+  setup({explicit_timeout: true});
+
+  const message1 = '1st';
+  const message2 = '2nd';
+  const message3 = new Uint8Array([51, 114, 100]);      // "3rd"
+  const message4 = new Uint8Array([52, 116, 104]);      // "4th"
+  const message5 = new Uint8Array([108, 97, 115, 116]); // "last"
+
+  const toUint8Array = buf => {
+    return buf instanceof ArrayBuffer ? new Uint8Array(buf) : buf;
+  }
+
+  // compare two ArrayBuffer or Uint8Array
+  const compare = (a, b) => {
+    const p = toUint8Array(a);
+    const q = toUint8Array(b);
+    return !!p && !!q && p.every((item, index) => { return item === q[index]; });
+  };
+
+  let connection;
+
+  const presentBtn = document.getElementById('presentBtn');
+  presentBtn.onclick = () => {
+    presentBtn.disabled = true;
+    const stash = new Stash(stashIds.toController, stashIds.toReceiver);
+    const request = new PresentationRequest('support/PresentationConnection_send_receiving-ua.html');
+
+    let eventWatcher;
+
+    promise_test(t => {
+      t.add_cleanup(() => {
+        if (connection.state === 'connecting') {
+          connection.onconnect = () => {
+            connection.terminate();
+          }
+        }
+        else if (connection.state === 'closed') {
+          request.reconnect(connection.id).then(c => {
+            c.terminate();
```

Comment:
This code assumes that `c` will be returned in a `connected` state, but the spec says it will be in a `connecting` state, so `terminate` won't have any effect (which seems to match what happens in Chrome at least). This should rather be `c.onconnect = () => { c.terminate(); }`.

FYI, I created w3c/presentation-api#421 to wonder whether we should not simplify the life of developers here.